schema/PublicationVolume.md

```diff
@@ -0,0 +1,60 @@
+| pageEnd               | `string` \| `integer`                                                           | The page on which the work ends; for example "138" or "xvi".                                                            | [PublicationVolume](./PublicationVolume.html) |
+| pageStart             | `string` \| `integer`                                                           | The page on which the work starts; for example "135" or "xiii".                                                         | [PublicationVolume](./PublicationVolume.html) |
+| pagination            | `string`                                                                        | Any description of pages that is not separated into pageStart and pageEnd; for example, "1-6, 9, 55" or "10-12, 46-49". | [PublicationVolume](./PublicationVolume.html) |
+| parts                 | `array<`​[`CreativeWork`](./CreativeWork.html)​`>`                              | Elements of the collection which can be a variety of different elements, such as Articles, Datatables, Tables and more. | [CreativeWork](./CreativeWork.html)           |
+| publisher             | [`Person`](./Person.html) \| [`Organization`](./Organization.html)              | A publisher of the CreativeWork.                                                                                        | [CreativeWork](./CreativeWork.html)           |
+| text                  | `string`                                                                        | The textual content of this creative work.                                                                              | [CreativeWork](./CreativeWork.html)           |
+| title                 | `string`                                                                        | The title of the creative work.                                                                                         | [CreativeWork](./CreativeWork.html)           |
+| url                   | `string<uri>`                                                                   | The URL of the item.                                                                                                    | [Thing](./Thing.html)                         |
+| version               | `string` \| `number`                                                            | The version of the creative work.                                                                                       | [CreativeWork](./CreativeWork.html)           |
+| volumeNumber          | `string` \| `integer`                                                           | Identifies the volume of publication or multi-part work; for example, "iii" or "2".                                     | [PublicationVolume](./PublicationVolume.html) |
+
+:::
+
+A part of a successively published publication such as a periodical or multi-volume work, often numbered. It may
+represent a time span, such as a year.
+
```

Comment:
In this introductory text it would be good to explain how `PublicationVolume` is usually used...to be able to refer to an `Article` or other `CreativeWork` in a bibliography. And link to the docs for those.